### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MaxSchmide**

### CODE REVIEW

**Review by cebanoleksandr — APPROVED** [review]
Good job:)

File: models/Expense.model.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+    allowNull: false,
+  },
+  userId: {
+    type: DataTypes.UUID,
+    allowNull: false,
+  },
```
Comment (anstsot) [inline models/Expense.model.js:16]: set the relation to `User` model

File: services/expenses.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  if (to) {
+    result = result.filter(({ spentAt }) => {
+      return Date.parse(spentAt) <= Date.parse(to);
+    });
+  }
+
+  return result;
+};
```
Comment (anstsot) [inline services/expenses.js:30]: better to create complex query instead of filtering all data

File: utils/db.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', '4426996VFRC', {
```
Comment (anstsot) [inline utils/db.js:6]: better to create constants
```suggestion
const DB_NAME = 'postgres';
//...

const sequelize = new Sequelize(DB_NAME, 'postgres', '4426996VFRC', {
```

File: controllers/expenses.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+const expensessService = require('../services/expenses.js');
+const usersService = require('../services/users.js');
+
+const getAll = (req, res) => {
+  const query = req.query;
+
+  if (!query) {
+    res.status(304).send('Not modified');
+  }
```
Comment (anstsot) [inline controllers/expenses.js:12]: just return all data instead

**Review by anstsot — CHANGES_REQUESTED** [review]
Good start:four_leaf_clover:
Let’s add a few fixes to improve your code.

**Pull request closed without merging**